{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"2:19:25 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"2:19:25 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"2:22:37 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"2:22:37 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"2:24:32 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"2:24:32 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"2:33:13 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"2:33:13 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"3:06:27 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"3:06:27 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"3:08:27 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"13.127.179.204:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:21:11)\n    at Server.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:319:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:309:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:177:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"13.127.179.204:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"4:14:16 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"13.127.179.204:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:21:11)\n    at Server.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:319:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:309:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:177:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"13.127.179.204:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"4:14:16 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"13.127.179.204:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:21:11)\n    at Server.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:319:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:309:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:177:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"13.127.179.204:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"4:14:17 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"13.127.179.204:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:21:11)\n    at Server.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:319:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:309:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:177:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"13.127.179.204:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"4:14:17 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"13.127.179.204:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:21:11)\n    at Server.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:319:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:309:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:177:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"13.127.179.204:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"4:14:17 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"13.127.179.204:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:21:11)\n    at Server.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:319:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:309:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:177:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"13.127.179.204:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"4:14:17 PM"}
{"level":"warn","message":"uncaughtException { DisconnectedError: Ran out of retries trying to reconnect to \"13.127.179.204:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.\n    at MongooseError.DisconnectedError (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\lib\\error\\disconnected.js:21:11)\n    at Server.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\lib\\drivers\\node-mongodb-native\\connection.js:64:31)\n    at emitOne (events.js:101:20)\n    at Server.emit (events.js:188:7)\n    at Pool.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\topologies\\server.js:319:16)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\pool.js:309:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (D:\\Office\\HorizonAPI\\SourceCode\\node_modules\\mongoose\\node_modules\\mongodb-core\\lib\\connection\\connection.js:177:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n    at Socket.emit (events.js:188:7)\n    at emitErrorNT (net.js:1278:8)\n    at _combinedTickCallback (internal/process/next_tick.js:74:11)\n    at process._tickCallback (internal/process/next_tick.js:98:9)\n  message: 'Ran out of retries trying to reconnect to \"13.127.179.204:27017\". Try setting `server.reconnectTries` and `server.reconnectInterval` to something higher.',\n  name: 'DisconnectedError' }","timestamp":"4:14:18 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"5:57:46 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"5:57:46 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:01:04 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:01:04 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:03:11 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:03:11 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:03:11 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:05:11 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:13:46 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:13:46 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:13:46 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:15:46 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:16:54 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:16:54 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:16:54 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:26:27 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:26:27 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:26:46 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:26:46 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:28:27 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:28:46 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:30:10 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:30:10 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:32:38 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:32:38 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:33:05 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:33:05 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:33:26 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:33:26 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:33:54 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:33:54 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:33:56 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:34:04 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:34:04 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:34:16 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:34:16 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:34:25 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:34:25 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:34:28 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:34:28 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:34:29 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:34:29 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:34:29 PM"}
{"level":"error","message":"LOG REFERER YTD http://localhost:4201/ ::1 string","timestamp":"6:34:47 PM"}
